src/transformStateWithClones.js

```diff
@@ -8,6 +8,34 @@
  */
 function transformStateWithClones(state, actions) {
   // write code here
+  const obj = { ...state };
+  const arrObj = [];
```

Comment:
Use better variable naming for example: result for array and stateCopy for state object